assets/src/util/operatorFormatting.tsx

```diff
@@ -0,0 +1,21 @@
+import { Vehicle } from "../realtime"
+import { joinTruthy } from "../helpers/dom"
+
+export const formatOperatorName = (
```

Comment:
**suggestion (non-blocking):** not strictly in scope for this PR, but I think this could also be used to get rid of some existing logic that does roughly the same thing in `propertiesList.tsx`.

Reply:
Also, it would be good to have tests for this new module as well.

Reply:
I wanted to do so, and I avoided it since this was getting big, I'll open a separate PR (or open a ticket otherwise).

Reply:
b215d090a4c82ac5219054f6a2c56a898f330e65